front/src/ionic-bottom-drawer/ionic-bottom-drawer.component.scss

```diff
@@ -0,0 +1,8 @@
+:host {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  left: 0;
+  z-index: 11 !important;
+  transform: translateY(100vh);
+}
```

Comment:
Github wants to see a new line at EOF :joy:

Reply:
ok nehm ich raus :+1: